src/app/introduction/activity-1/activity-1.component.ts

```diff
@@ -69,8 +71,10 @@ export class HighlightActivityComponent implements OnInit {
                 });
                 this._sharedData.customSuccessAlert(this.alerts.activitySuccessMsg, this.alerts.activitySuccessTitle);
                 this.activityComplete = true;
-                if (!this.completed) { const currStage = 1;
-                  this._performanceService.openDialog(currStage); }
+              if (!this.completed) {
+                this._performanceService.openDialog(this.currStage);
+                this._leaderBoardService.updateLeaderBoard({activity: 'highlightDefinition', level: 'level1'})
```

Comment:
@shreyans29 I changed the hardcoded string of activities in all the files.
But, since **level** is dynamic and would depend on different levels, so this would become a global variable when we would have **muliple levels** PR merged.